docs/sources/write/style-guide/word-list/index.md

```diff
@@ -82,6 +82,9 @@ Use this rather than _hamburger menu_ or _kebab menu_.
 
 ## R
 
+### React
+
+Spell as React, not React.js or ReactJS.
```

Comment:
I'm curious... why not ReactJS? Thsi makes it clear you're referring to the JS library vs the ReactNative framework, and ReactJS is very commonly used within the JS dev community.

Reply:
> I'm curious... why not ReactJS? This makes it clear you're referring to the JS library vs the ReactNative framework, and ReactJS is very commonly used within the JS dev community.

My reasoning is that react.dev, the official site, says the name is React; also Wikiipedia uses React. Other versions of the name exist, and hence there is the potential for inconsistencies.

Reply:
Also I don't think anyone in Grafana uses React Native for anything.